src/main/java/Consolidator.java

```diff
@@ -154,6 +154,14 @@ protected void processLine(String line) {
         			currentBlock.requireNonEmptySource = true;
         			return;
         		}
+        		if(row[1].equals("treatFormulaAsBlank")) {
+        			currentBlock.treatFormulaAsBlank = true;
+        			return;
+        		}
+        		if(row[1].equals("autoSkipOnError")) {
```

Comment:
is this lookup index correct? both keys, `treatFormulaAsBlank` and `autoSkipOnError`, are accessing the 2nd index of row. Moreover, I would use an `else if` statement for the 2nd if.